src/util/fabric/theme.ts

```diff
@@ -1,65 +1,42 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 
 /**
- * Override Fabric's theme variables. Note this this window.FabricConfig is temporary,
- * soon Fabric will have a loadTheme() utility to handle this.
+ * Override Fabric's theme variables.
  */
 
-import { loadTheme, IRawStyle } from 'office-ui-fabric-react/lib/Styling';
+import { loadTheme } from 'office-ui-fabric-react/lib/Styling';
+import { theme } from '../colors';
 
-const fontStyles: IRawStyle = {
-  color: '#343A41',
-};
-
-// NOTE: Would be handy to have access to our CSS variable values in JS...
 loadTheme({
-  palette: {
-    themePrimary: '#6c98d9',
-    neutralSecondary: '#646d7a',
-    neutralTertiary: '#dde0e6',
-    neutralLighter: '#f3f5f8',
-    neutralDark: '#dde0e6',
-    redDark: '#d1423b',
-  },
+  palette: theme.palette,
   fonts: {
-    tiny: fontStyles,
     xSmall: {
-      ...fontStyles,
       fontSize: '1.0rem',
       lineHeight: '1.6rem',
     },
     small: {
-      ...fontStyles,
       fontSize: '1.2rem',
       lineHeight: '1.6rem',
     },
     smallPlus: {
-      ...fontStyles,
       fontSize: '1.4rem',
       lineHeight: '2.0rem',
     },
     medium: {
-      ...fontStyles,
       fontSize: '1.5rem',
       lineHeight: '2.0rem',
     },
-    mediumPlus: fontStyles,
     large: {
-      ...fontStyles,
       fontSize: '1.8rem',
       lineHeight: '2.4rem',
     },
     xLarge: {
-      ...fontStyles,
       fontSize: '2.2rem',
       lineHeight: '2.8rem',
     },
     xxLarge: {
-      ...fontStyles,
       fontSize: '2.4rem',
       lineHeight: '2.8rem',
     },
-    superLarge: fontStyles,
-    mega: fontStyles,
```

Comment:
Nice.

Reply:
Weird thing is that it looks like these fontStyle colors had no effect.

Reply:
these did override the default typography element colors. are visual diffs still passing?

Reply:
Yes, the visual diffs still passed.  I manually inspected some elements, but had trouble finding the rule being applied anywhere.

Looking closer now at `Block`, I see the color is being overridden by a merge style, so instead of `#343A41`  it is `rgb(51, 51, 51)`.

I'll look a little closer at this, and see if our visual diff picks up the difference.